R/pxweb_split_query.R

```diff
@@ -23,8 +23,10 @@ pxweb_split_query <- function(pxq, px, pxmd){
   pxqd <- pxweb_query_dim(pxq)
   # Get variables that can be split
   pxqds <- pxweb_query_dim_splittable(pxq, pxmd)
-  mxv <- px$config$max_values_to_download
   
+  mxv <- px$config$max_values_to_download
+  .GlobalEnv$mxv <- mxv
```

Comment:
We should not write to the global environment. I think then this will not pass CRAN checks.

Reply:
Thank you for your comment. Although it was neither a warning nor an error, the check produced the following NOTE when the variable mxv is created in pxweb_split_query function and referred to in split_dimensions_left_right function:

```
checking R code for possible problems ... NOTE
  split_dimensions_left_right: no visible binding for global variable
    ‘mxv’
  Undefined global functions or variables:
    mxv
```

I will see what would be a proper way to handle this.

Reply:
Ah ok. I think we want to add it the correct way then. If you want, leave the NOTE and Ill fix it when I submit to CRAN.